homeassistant/components/device_tracker/ble_tracker.py

```diff
@@ -0,0 +1,90 @@
+"""Tracking for bluetooth devices."""
+import logging
+from datetime import timedelta
+
+from homeassistant.helpers.event import track_point_in_utc_time
+from homeassistant.components.device_tracker import (
+    YAML_DEVICES,
+    CONF_TRACK_NEW,
+    CONF_SCAN_INTERVAL,
+    DEFAULT_SCAN_INTERVAL,
+    load_config,
+)
+import homeassistant.util as util
+import homeassistant.util.dt as dt_util
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['gattlib']
+
+BLE_PREFIX = 'BLE_'
+
+def setup_scanner(hass, config, see):
+    """Setup the BLE Scanner."""
```

Comment:
I am still confused. The other bluetooth tracker depends on pybluez. pygattlib claims to use the gatttool from Bluez.

Let's call it bluetooth_le_tracker.py, that way it is more clear it's still bluetooth. We should also make sure to emphasize the differences in the docs.